### PR TITLE
hotfix to only render toolbar once

### PR DESCRIPTION
**Pull request by bryevdv**
Fixes a bug that caused the toolbar to be unresponsive on streaming updates. 

@birdsarah @mattpap @brendancol any comments welcome, I am sure this is a simplistic fix but it does seem to be an improvement.

### CODE REVIEW

**Comment by bryevdv** [comment]
@damianavila this would be good to get in the dev build before the webinar

**Comment by birdsarah** [comment]
I've tested locally on a few normal plots and on the server plots. The fix works as expected. It's so cool being able zoom in and pan on the animated plot.

File: bokehjs/src/coffee/common/tool_manager.coffee [hunk 1]
```diff
@@ -16,9 +16,12 @@ class ToolManagerView extends Backbone.View
   initialize: (options) ->
     super(options)
     @listenTo(@model, 'change', @render)
```
Comment (mattpap) [inline bokehjs/src/coffee/common/tool_manager.coffee]: This PR makes this line pointless.

**Comment by mattpap** [comment]
I think this PR doesn't fix the underlying rendering issue. Needs more thought (I don't mind doing this myself). -1 for now.

**Comment by bryevdv** [comment]
@mattpap OK, the dev build for the webinar is this afternoon...

**Comment by mattpap** [comment]
> OK, the dev build for the webinar is this afternoon...

I see, so lets merge this for now.

**Comment by mattpap** [comment]
Obviously this will need a followup.

**Comment by bryevdv** [comment]
Agreed needs followup, the only reason I'm OK for a quick fix is that to my knowledge no one has ever asked or demonstrated updating the toolbar tools after a plot is drawn.

**Pull request merged**